tests/plugins/inlinetoolbar/manual/kama.html

```diff
@@ -0,0 +1,61 @@
+<style>
+	body {
+		margin-left: 0px;
+		padding-left: 400px;
+	}
+</style>
+<textarea id="editor1" cols="10" rows="10">
+	<p>Apollo 11</p>
+	<p>Apollo 11</p>
+	<p>Apollo 11</p>
+	<img src="http://lorempixel.com/400/200/">
```

Comment:
Please use local images e.g. `tests/_assets/` as the remote ones may not be always available (or load slow).